wiki/releasenotes3/tfgrid_release_3_0_a5.md

```diff
@@ -0,0 +1,86 @@
+
+However, TFT is still the native currency on TFChain. As such, there is no need for an external service to transfer tokens on TFChain. A transaction fee is charged (currently 0.01 TFT) for every transaction/extrinsic call. 
+
+## TF Farm Management Tool v3.0.2
+
+### Adjust farm management for the latest TFChain upgrade
+
+On the last release of 3 alpha-4, we have released Farm Management Tool v3.0.1 which allows farmers to migrate their farms from v2 to v3 through the TF Connect application.
+
+Recently a change was made within the codes of TFChain, thus broke the farm management function in the wallet. Therefore, on the v3.0.2 release, code changes were done on **Farm Management Tool to adapt itself with TFChain changes**. This was done quickly in production.
+
+## TFConnect App v3.5.0
+
+### Generic Frontend and Backend Improvements
+This new feature release of TFConnect App includes new features such as **enabling user to sign documents on the app** directly, and many other backend improvements.
+
+### Integrate TFConnect SSO to TFPlay Solutions
+
+We need to simplify peer-to-peer collaboration and how users interact with their TFPlay solutions. On this release, we have eliminated a complicated way of signing up to solutions (emails, username and password) by replacing it using **TFConnect app SSO login**. Therefore on this release, we successfully created TFConnect Native SSO backend environments for the following TFPlay solutions**: Discourse. Mattermost, and Gitea, that will allow users to sign in and start using the solutions with just few clicks.
+
+## TFChain v1.2.0
+
+### ThreeFold DAO Pt. 2: Adjoint Validator-Council member request
+
+From version 3.0 on, [ThreeFold Grid operates as a DAO](https://library.threefold.me/info/threefold#/tfgrid/threefold__dao). On the last release of TFChain v1.0.0,  we have successfully implemented The first TF-DAO that allows users to request to become DAO council members. 
+
+On this release we also have successfully implemented **ThreeFold DAO Request part 2 where any user could request to become an adjoint validator-council member**: by running a a validator node, not only that they become a validator, they would also gain a seat as DAO council member that give them the right to vote for organizational changes.
+
+### ThreeFold DAO Pt. 2: Enable Validator Application
+
+On this release we implemented ThreeFold DAO Request part 2 any user could **apply to become a validator **and register the validator application on-chain if they meet the validator requirements.
+
+## TFGrid Client Proxy v1.2.0
+
+### Implement REST API
+
+TFGrid Client Proxy contains list of farms, nodes and node details information on the TFGrid. On this release of TFGrid Client Proxy, we have implemented REST API:  a set of application integration protocols that follows the **REST- (REpresentational State Transfer)** architectural style for distributed hypermedia systems. 
```

Comment:
remvoe everything after `have implemented REST API` and Add a note that We plan to stop exposing graphql servers to the public too)